### PR TITLE
feature: Add affiliation when a new user registers

### PR DESCRIPTION
**Pull request by DanielPalafox**

### CODE REVIEW

File: orcid-core/src/main/java/org/orcid/core/manager/impl/RegistrationManagerImpl.java [hunk 1]
```diff
@@ -290,7 +297,12 @@ private String createMinimalProfile(Registration registration, boolean usedCaptc
             name.setGivenNames(new GivenNames(registration.getGivenNames().getValue()));
         }
         recordNameManager.createRecordName(orcid, name);
-                
+
+        if (Features.EVENTS.isActive() && registration.getAffiliationForm() != null) {
```
Comment (amontenegro) [inline orcid-core/src/main/java/org/orcid/core/manager/impl/RegistrationManagerImpl.java:301]: @DanielPalafox is this the right feature for this `Features.EVENTS`? I think this one is to track user events right?

File: orcid-web/src/main/java/org/orcid/frontend/web/controllers/RegistrationController.java [hunk 2]
```diff
@@ -321,6 +329,28 @@ public void validateRegistrationFields(HttpServletRequest request, Registration
         regEmailValidate(request, reg, false, false);
         registerTermsOfUseValidate(reg);
 
+        if (Features.EVENTS.isActive() && reg.getAffiliationForm() != null) {
```
Comment (amontenegro) [inline orcid-web/src/main/java/org/orcid/frontend/web/controllers/RegistrationController.java:332]: @DanielPalafox again, I think the feature `Features.EVENTS` is not the one we want to use here, could you please create one, something like `Features.AFFILIATIONS_ON_REGISTRATION`

**Review by amontenegro — CHANGES_REQUESTED** [review]

File: orcid-core/src/main/java/org/orcid/core/manager/impl/RegistrationManagerImpl.java (hunk 1 above)
Comment (DanielPalafox) [reply orcid-core/src/main/java/org/orcid/core/manager/impl/RegistrationManagerImpl.java]: Yes, you are right I will put the correct togglz. Thanks

File: orcid-web/src/main/java/org/orcid/frontend/web/controllers/RegistrationController.java (hunk 2 above)
Comment (DanielPalafox) [reply orcid-web/src/main/java/org/orcid/frontend/web/controllers/RegistrationController.java]: Done

**Review by amontenegro — APPROVED** [review]

**Pull request merged**